politeiawww/user.go

```diff
@@ -200,6 +205,56 @@ func validatePubkey(publicKey string) ([]byte, error) {
 	return pk, nil
 }
 
+// formatName normalizes a contractor name to lowercase without leading and
+// trailing spaces.
+func formatName(name string) string {
+	return strings.ToLower(strings.TrimSpace(name))
+}
+
+func validateName(name string) error {
+	if len(name) < www.PolicyMinUsernameLength ||
+		len(name) > www.PolicyMaxUsernameLength {
+		log.Tracef("Name not within bounds: %s", name)
```

Comment:
Can you change these `Tracef` statements to `Debugf` statements?  This is something that we've been inconsistent about in the past.  Some parts of politeiawww use `Tracef` and some use `Debugf` for user errors.  Moving forward though, I'd like to use `Debugf` for logging user errors and `Tracef` for tracing function calls.